docs/journey/day1/README.md

```diff
@@ -174,7 +176,7 @@ The`webhook.secret` is used to authenticate incoming hooks from Git host.
 
 ## Bringing the bootstrapped environment up
 
-First of all, let's get started with our Git repository.
+Ignore these steps if the flag `--push-to-git=true` is part of your bootstrap command.
```

Comment:
This isn't quite true, we need to bootstrap the ArgoCD application the first time, which causes the rest of the configuration to be fetched from the GitOps repository.

Reply:
I think this question and the conversation above come from how one interprets "Bringing the bootstrapped environment up".   I think it should be re-worded to "Pushing the bootstrapped environment to Git" to address the confusion.

Reply:
```--push-to-git=true``` by default initialize the local gitops dir and push it into github 
```
$ kam bootstrap --service-repo-url https://github.com/amitkrout/taxi --gitops-repo-url https://github.com/amitkrout/gitops.git --image-repo quay.io/amitkrout/taxi --dockercfgjson kam-bot-kambot-auth.json --git-host-access-token ******************* --output resources --push-to-git=true
  
Checking dependencies

 ✓  Checking if Sealed Secrets is installed with the default configuration [908ms]
 ✓  Checking if ArgoCD is installed with the default configuration [2s]  
 [...]
 Next Steps:
 Please refer to https://github.com/redhat-developer/kam/tree/master/docs to get started.

$ cd resources/

$ ls -a
.               .git            environments
..              config          pipelines.yaml
```